### ISSUE TITLE
Review: 2.4.13 Focus Appearance (AAA)

### ISSUE THREAD

**Issue by qbalsdon** [GetEvinced/mcag#61]
Task: Edit [MCAG](https://github.com/GetEvinced/mcag/blob/main/src/app/data/mcag.json) section 2.4.13 considering the mobile research questions (to be completed)

---
When the keyboard focus indicator is visible, an area of the focus indicator meets all the following:

    is at least as large as the area of a 2 CSS pixel thick perimeter of the unfocused component or sub-component, and

    has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.

Exceptions:

    The focus indicator is determined by the user agent and cannot be adjusted by the author, or

    The focus indicator and the indicator’s background color are not modified by the author.

Note 1: What is perceived as the user interface component or sub-component (to determine enclosure or size) depends on its visual presentation. The visual presentation includes the component's visible content, border, and component-specific background. It does not include shadow and glow effects outside the component's content, background, or border.

Note 2: Examples of sub-components that may receive a focus indicator are menu items in an opened drop-down menu, or focusable cells in a grid.

Note 3: Contrast calculations can be based on colors defined within the technology (such as HTML, CSS and SVG). Pixels modified by user agent resolution enhancements and anti-aliasing can be ignored.
---
Sufficient Techniques for Success Criterion 2.4.13

Note: Other techniques may also be sufficient if they meet the success criterion. See [Understanding Techniques.](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140916/understanding-techniques.html#understanding-techniques)

    G195: Using an author-supplied, visible focus indicator
    C40: Creating a two-color focus indicator to ensure sufficient contrast with all components
    [C41: Creating a strong focus indicator within the component ](https://www.w3.org/WAI/WCAG22/Techniques/css/C41.html)

Failures for Success Criterion 2.4.13

    Using a CSS border for inline text which can wrap (Potential future technique)
